src/displayBackend/DisplayCommandHandler.cpp

```diff
@@ -175,11 +165,7 @@ void DisplayCommandHandler::createDisplayBuffer(const xendispl_req& req)
 	 * is not supported with a non-zero offsets: do not let us down to
 	 * creating any resources and terminate the command here.
 	 */
-#if (XENDISPL_PROTOCOL_VERSION >= 2)
```

Comment:
+1 for the change. But please fix the comment above as well.

Also, I would add mention about compatibility. Something like "V1 declares this field as reserved with default value 0. This is perfectly compatible with V2, because it will see offset 0, which is totally correlates with the offset used by V1"

Reply:
makes sense

Reply:
```
 /*******************************************************************************
 * Protocol differences between its versions and their implications
 *******************************************************************************
 * Version 2
 *******************************************************************************
 * - XENDISPL_OP_DBUF_CREATE introduces data offset field: if frontend doesn't
 *   support data offset field, e.g. it uses protocol version 1, then
 *   according to the protocol it sees this field as "reserved" which must
 *   be set to 0, thus effectively meaning for the backend zero offset is used.
 * - XENDISPL_OP_GET_EDID is an optional request, so if frontend uses protocol
 *   version 1 it is not known to it, thus it is not going to be issued.
 ******************************************************************************/

```
I have added the comments in the corresponding commits